src/priiloader/hacks_hash.ini

```diff
@@ -249,6 +249,45 @@ offset=0x8137ad0c
 patch=0x4bf840f4
 offset=0x812fef00
 patch=0x815488f8,0x8137ad10,0x00000000,0x00000000
+[Wiimmfi Patch v4 for vWii 5.2.0J]
+maxversion=608
+minversion=608
+amount=4
+require=wiimmfi-v4
+offset=0x8137be44
+patch=0x4bf831ec
+offset=0x812ff000
```

Comment:
why offset's and not hashes?

Reply:
First one is offset because both the offset and the patch value are different for each system menu anyways, and the whole patch is generated by a script. Just putting the offset is easier and less error-prone than a hash. The build script for the hack needs to have the offsets hardcoded anyways to be able to calculate the branches, so there's no harm in putting them into the hack.
Second one is offset because the space (812ff000) is empty space in the system menu so there's no way to "access" that space using a hash because there's nothing to search. That was the discussion we already had in #218.